app/src/page/file/view.js

```diff
@@ -512,7 +593,7 @@ class FilePage extends React.PureComponent {
                     <FileItemMedia style={filePageStyle.thumbnail} title={title} thumbnail={thumbnail} />}
                   {((!this.state.downloadButtonShown || this.state.downloadPressed) && !this.state.mediaLoaded) &&
                       <ActivityIndicator size="large" color={Colors.LbryGreen} style={filePageStyle.loading} />}
-                  {((isPlayable && !completed && !canLoadMedia) || !completed || canOpen) && (!this.state.downloadPressed) &&
+                  {((isPlayable && !completed && !canLoadMedia) || canOpen || (!completed && !this.state.streamingMode)) && (!this.state.downloadPressed) &&
```

Comment:
Maybe we should show the download button always? Or if the video is playing? In this design, is it now impossible to _not_ stream a video? What if I want to save for offline viewing?

Reply:
The initial download / view button will show up if:
* the `fileInfo` is present and it's a playable media file (audio / video)
* it's not a media file, but it is a viewable file: `canOpen` (image, text/plain, text/html, text/*)
* and the user has not pressed it yet.

Once the user has pressed it, we begin the download or stream and then the file page has to display one of:
* an activity indicator (download is about to start or media cannot be played yet)
* the media player (audio / video) - `isPlayable` and `canLoadMedia` are `true`, or `this.state.streamingMode` is `true` if not downloading. 
* the image viewer (images) - `completed` and `canOpen` are `true`
* the web viewer (text/* types) - same for images
* the same download button (not interactable) showing the download progress for other file types.